### DISCUSSION TITLE
BO score errors

### DISCUSSION THREAD

**Discussion by tw-mat** [Tomoki-YAMASHITA/CrySPY#74] [Q&A]
Many thanks in advance. I came across the following errors in the BO scores when using CrySPY-1.2.5 in combination with the Combo3 library. It seems the FP descriptors and the predicted function have been obtained but Thompson sampling failed to calculate the BO scores. I used very basic settings for BO (please see below). 

```
> BO_status
  Struc_ID         Score          Mean      Variance
         1    6.18672804    6.52748614    0.00067417
         2           nan    6.59392153    0.00041170
         3           nan    6.68873308    0.00124214
         5           nan    6.56890051    0.00095725
         6           nan    6.59149058    0.00042065

> cryspy.in
[BO]
nselect_bo = 100
score = TS
dscrpt = FP
```

**Comment by Tomoki-YAMASHITA** [comment]
I have no idea why this error occurred. If you don't have a preference, try using a different score such as EI.
